openvdb/Makefile

```diff
@@ -822,21 +823,21 @@ pdfdoc:
 	@echo "$@"': $$DOXYGEN is undefined'
 endif
 
-vdb_lod: $(LIBOPENVDB) cmd/openvdb_lod/main.cc
+vdb_lod: $(LIBOPENVDB) cmd/openvdb_lod.cc
```

Comment:
I find it slightly obscure that "openvdb_lod.cc" produces "vdb_lod". Could we perhaps rename to vdb_lod.cc so at least the names are consistent (and likewise for all the other binaries)?

Reply:
I guess, though "openvdb" is the prefix used for everything else.  The reason I didn't name the commands `openvdb_print`, `openvdb_view`, etc. is that I figured users would rebel if they had to type those extra four characters all the time.

Reply:
We could build openvdb_* binaries and also install symlinks to their vdb_* aliases

Reply:
I'm going to punt on the file naming issue for now.  Let's revisit it after the CMake switchover.